daemons/execd/execd_commands.c

```diff
@@ -287,7 +287,8 @@ build_rsc_from_xml(xmlNode * msg)
     rsc->work = mainloop_add_trigger(G_PRIORITY_HIGH, lrmd_rsc_dispatch, rsc);
 
     // Initialize fence device probes (to return "not running")
-    rsc->fence_probe_result = PCMK_EXEC_NO_FENCE_DEVICE;
+    pcmk__set_result(&rsc->fence_probe_result, CRM_EX_ERROR,
```

Comment:
When it is some kind of initialization we might consider introduction of more specific
initializers than PCMK__UNKNOWN_RESULT like PCMK__NO_FENCE_DEVICE_RESULT.
Don't know if it is worth it as it of course has to be used with care (e.g. not below) as not
to overwrite a string-pointer.